src/client/components/templates/LoginTemplate/LoginTemplate.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ShapesContainer from '../../organisms/ShapesContainer/ShapesContainer';
+import LoginForm from '../../organisms/Login-form/LoginForm';
+
+import './LoginTemplate.scss';
+
+const LoginTemplate = (props) => {
+  return (
+    <div className='login-template'>
+      <ShapesContainer />
+      <LoginForm {...props} />
```

Comment:
Please avoid props propagation, this is wonderful but it can make the code harder to read comparing with knowing exactly which prop is passing over here.

Reply:
Done